e2e/app/component/copy-modal.ts

```diff
@@ -2,9 +2,10 @@ import Modal from 'app/component/modal';
 import Textbox from 'app/element/textbox';
 import {LinkText} from 'app/text-labels';
 import {ElementHandle, Page} from 'puppeteer';
+import {waitWhileLoading} from 'utils/test-utils';
 
 
-export default class ConceptsetCopyModal extends Modal {
+export default class CopyModal extends Modal {
```

Comment:
The `Copy Modal` window UI is same for copy Concept Set and Notebook. I renamed class name to make it reusable/generic.